runtest.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cd "$(dirname "${1}")"
+diff "${1}" <("${2}")
```

Comment:
How does this actually work? I guess it's running something like `diff baseline_multi.txt`, but what's the `<(...)` redirection for?

Reply:
Yup, that's executing the command in the second argument, piping its stdout to a named pipe, and then comparing those contents against the baseline file.  I got this trick from [this answer](https://unix.stackexchange.com/a/155807) (but only one of the inputs needs to be a command instead of both)